VTEX - Marketplace APIs - Suggestions.json

```diff
@@ -511,16 +511,36 @@
                         "schema": {
                             "type": "string",
                             "default": "application/json"
-                        }
+                    }
                     }
                 ],
                 "responses": {
                     "200": {
                         "description": "OK",
-                        "headers": {}
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/responseGetSellersApprovalSettings"
+                                },
+                                "example": {
+                                    "sellerId": "sellerexample",
+                                    "accountId": "d74dau71f-325a-4463-bd53-ae8b0453186ca",
+                                    "accountName": "marketplaceexample",
+                                    "mapping": {
+                                        "defaultColor": "defaultColor",
+                                        "searchColor": "",
+                                        "color1": "black",
+                                        "color2": "white",
+                                        "color": "black",
+                                        "size": "sizeproduct"
+                                    },
+                                    "matchFlux": "Default"
+                                }
+                            }
+                        },
+                        "deprecated": false
```

Comment:
Esse erro deve ocorrer por ter alguma chave, bracket ou vírgula faltando, ou sobrando, aí o arquivo não interpreta o `deprecated` como estando no lugar certo. Checa se tudo que abre, fecha com seu par adequado

`Property "deprecated" is not expected to be here.`